torchaudio/datasets/vctk.py

```diff
@@ -123,9 +122,7 @@ def __init__(self,
                 "with `download=True` to donwload the latest version."
             )
 
-        walker = walk_files(
-            self._path, suffix=self._ext_audio, prefix=False, remove_suffix=True
-        )
+        walker = sorted(str(p.stem) for p in Path(self._path).glob('*/*/*' + self._ext_audio))
```

Comment:
Can you change the pattern to `'**/*' + self._ext_audio`?

That should match the behavior of the original implementation.